pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -219,7 +219,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarListeners(env *model.Environme
 			buildSidecarInboundListeners(configgen, env, node, push, proxyInstances).
 			buildSidecarOutboundListeners(configgen, env, node, push, proxyInstances).
 			buildManagementListeners(configgen, env, node, push, proxyInstances).
-			buildVirtualOutboundListener(env, node).
+			buildVirtualOutboundListener(configgen, env, node, push, proxyInstances).
 			buildVirtualInboundListener(env, node)
```

Comment:
what about the inbound listener? It too has blackhole clusters

Reply:
AFAICT `buildVirtualInboundListener` will only add `Passthrough` for the inbound listener.

Reply:
In the inbound direction, i'm not sure adding stats make sense. Shouldn't we always the listener with POD_IP_POD_PORTS after original destination is used on the inbound listener?

Reply:
I don't think it is used and doesn't make sense to me. Currently only Passthrough is configured for inbound virtuak listener.